src/display/network.js

```diff
@@ -414,9 +415,27 @@ PDFNetworkStreamFullRequestReader.prototype = {
       networkManager.abortRequest(fullRequestXhrId);
     }
 
+    // Content-Disposition: attachment; filename=Naïve file.txt
+    var fullRequestXhr = networkManager.getRequestXhr(fullRequestXhrId);
+    var contentDisposition =
+      fullRequestXhr.getResponseHeader('Content-Disposition');
+    this._fileName = this._extractNameFrom(contentDisposition);
+
     this._headersReceivedCapability.resolve();
   },
 
+  _extractNameFrom:
```

Comment:
Let's change this line and the next line to simply `_extractNameFrom(contentDisposition)`. We use anonymous functions a lot and it makes it easier to read (also writing `contentDisposition` instead of `cDisposition` is more readable).

I think it may also be more clear to rename this to `_extractNameFromHeader` to be more explicit about where it's extracted from.